components/Report/Chart/ReportBarChart.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+import { Bar, BarChart, CartesianGrid, Tooltip, XAxis, YAxis } from 'recharts';
+import styles from '../Report.module.css';
+
+type ReportBarChartProps = {
+    title: string;
+    data: Array<{ name: string; value: number; fill: string }>;
+};
+
+export default function ReportBarChart(props: ReportBarChartProps) {
+    const { title, data } = props;
+    return (
+        <div className={styles.outerBarChartContainer}>
+            <h2>{title}</h2>
+            <div className={styles.innerBarChartContainer}>
+                <div>
+                    {data.map((dataPoint: any) => (
+                        <p key={dataPoint.name}>
+                            {dataPoint.name}: {dataPoint.value}
+                        </p>
+                    ))}
+                </div>
+                <div style={{ flex: 1 }}>
+                    <BarChart
+                        width={700}
```

Comment:
We could consider having this fill the available width instead of a fixed 700

Reply:
added responsive container!